swarm/pss/pss.go

```diff
@@ -365,17 +371,25 @@ func (self *Pss) process(pssmsg *PssMsg) bool {
 			self.outbox <- pssmsg
 		}()
 	}
-	handlers := self.getHandlers(psstopic)
+	if psstopic == rawTopic {
```

Comment:
so lets talk about this. I dont understand why for no encryption there would be a specific topic rather than a bool field on the message or nothing (just the first assumption)

Reply:
_nothing_ sounds better. Let's try that.